uniffi_bindgen/src/interface/types/mod.rs

```diff
@@ -63,9 +64,16 @@ pub enum Type {
     Sequence(Box<Type>),
     Map(/* String, */ Box<Type>),
     // An FfiConverter we `use` from an external crate
-    External { name: String, crate_name: String },
+    External {
+        name: String,
+        crate_name: String,
+    },
     // A local type we will generate an FfiConverter via wrapping a primitive.
-    Wrapped { name: String, prim: Box<Type> },
+    Wrapped {
+        name: String,
+        prim: Box<Type>,
```

Comment:
Nit: prefer `primitive_type` or `lower_type`

Reply:
Note these names are already in use, and just being reformatted - I don't think it's reasonable for this PR to rename existing names. Also, FWIW, Rust really prefers shorter names, so I think `prim` is perfectly reasonable - could maybe agree `primitive`, but strongly -1 against `primitive_type: Box<Type>` :) I even considered using `nm` instead of `name` which already appears in many places :)